interpreter/text/parser.mly

```diff
@@ -1062,13 +1086,26 @@ module_var_opt :
 
 module_ :
   | LPAR MODULE module_var_opt module_fields RPAR
-    { $3, Textual ($4 (empty_context ()) () @@ at ()) @@ at () }
+    { let m = $4 (empty_context ()) () @@ at () in
+      $3, Textual (m, parse_annots m) @@ at () }
 
 inline_module :  /* Sugar */
-  | module_fields { Textual ($1 (empty_context ()) () @@ at ()) @@ at () }
+  | module_fields
+    { let at = at () in
+      (* Hack for empty modules *)
+      let at = if at.left <> at.right then at else
+        {at with right = {at.right with line = Stdlib.Int.max_int}} in
```

Comment:
If only I remembered what this was needed for...

Reply:
tests like this one, with an empty module (just an annotation) fail without those lines:

```
(module quote "(@a \09)")  ;; \t
```